ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -187,18 +189,29 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
     public incrementStep(scroll: boolean = true): void {
```

Comment:
I haven't checked other studies, but I recall that we've overridden `incrementStep()` in RGP study. Do we need to apply same change there?

Reply:
Seems like `ATCP` and `RGP` studies override the method.
The bug issue occurred for an activity with an obligatory mailing address. 
If `RGP` has such a case, I guess, it is worth applying the changes.

Reply:
@MocanaAtBroad, what do you think ^?

Reply:
Let's leave at alone as much as possible. I am seeing they did all kinds of stuff to get things to do what they wanted.